ui/src/components/DeleteModal/DeleteModal.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { render, screen } from '@testing-library/react';
+import React, { createContext } from 'react';
+import userEvent from '@testing-library/user-event';
+import { AxiosResponse } from 'axios';
+import * as axiosWrapper from '../../util/axiosCallWrapper';
+import DeleteModal from './DeleteModal';
+import { setUnifiedConfig } from '../../util/util';
+import { UnifiedConfig } from './DeleteModal.stories';
+
+jest.mock('immutability-helper', () => {
+    return () => 1;
+});
+
+describe('Text Area Component', () => {
+    const handleClose = jest.fn();
+
+    const TableContext = createContext({
+        rowData: { serviceName: { stanzaName: 1 } },
+        setRowData: () => {},
+    });
+
+    beforeEach(() => {
+        setUnifiedConfig(UnifiedConfig);
```

Comment:
mocking whole util after imports is enough to mock global config
```ts
jest.mock('../../util/util');

Reply:
true, changed